[Changelog] 25.1.0_cycle5.mdx (#521)
seqeralabs/docs: changelog/seqera-cloud/25.1.0_cycle5.mdx

Affected endpoints: 
- cr.seqera.io
- *.connect.cloud.seqera.io
- intern.seqera.io
- ai.seqera.io
- hub.seqera.io
- cloud.seqera.io
- api.cloud.seqera.io
- user-data.cloud.seqera.io
- tower.nf
- api.tower.nf
- public.cr.seqera.io
- auth.cr.seqera.io
- wave.seqera.io
- licenses.seqera.io
- licman.seqera.io
- api.multiqc.info
:::

### Feature updates and improvements

- **Studios**
  - Private sessions: When adding a new session, the number of concurrent connections can now be defined in **General Config > Collaboration**.
  - Dynamic Studio templates: Labels have been added to indicate the status of support (e.g., *Recommended*, *Deprecated*) for a Studio template version. Users can now also switch the Studio's base template image when starting a session as new.
- **Nextflow**
  - Update `WfManifest` class to include the new `icon` property introduced in Nextflow.
  - Add a new migration file to add the new icon column into all the tables of entities that use the `WfManifest` class as @Embedded property.

> - Audit log update: Publish a new *pipeline_updated* audit event whenever a user updates a pipeline.

Reviewer: ```suggestion
- Audit log update: Pipeline edit events are now logged.
```

---
title: Seqera Cloud v25.1.0_cycle5
date: 2025-03-19
tags: [seqera cloud]
---

:::note
From Wednesday, February 26, 2025, Seqera endpoints no longer support Windows XP and Safari versions 8 and below (including OSx 10.10 or older). This change is necessary to ensure the security and performance of our services. We recommend updating your operating system and browser version to ensure uninterrupted access to our services.

Affected endpoints: 
- cr.seqera.io
- *.connect.cloud.seqera.io
- intern.seqera.io
- ai.seqera.io
- hub.seqera.io
- cloud.seqera.io
- api.cloud.seqera.io
- user-data.cloud.seqera.io
- tower.nf
- api.tower.nf
- public.cr.seqera.io
- auth.cr.seqera.io
- wave.seqera.io
- licenses.seqera.io
- licman.seqera.io
- api.multiqc.info
:::

### Feature updates and improvements

- **Studios**

>   - Private sessions: When adding a new session, the number of concurrent connections can now be defined in the **General Config > Collaboration**.

Reviewer 2: ```suggestion
  - Private sessions: When adding a new session, the number of concurrent connections can now be defined in **General Config > Collaboration**.
```

>   - Dynamic studio templates: Labels have been added to indicate the status of support (e.g., *Recommended*, *Deprecated*) for a Studio template version. Users can now also switch the Studio's base template image when starting a session as new.

Reviewer: ```suggestion
  - Dynamic Studio templates: Labels have been added to indicate the status of support (e.g., *Recommended*, *Deprecated*) for a Studio template version. Users can now also switch the Studio's base template image when starting a session as new.
```

> - **Seqera Compute**

Reviewer: Should we be showing Seqera Compute entries already? It's not even in Private preview yet. AFAIK this won't be available in prod. @swampie to confirm.

  - Updated Seqera Compute compute environment creation form with new fields: Advanced options, Pre/post run scripts, Nextflow config.
  - Added a **Features** tab to the admin workspace management page and a toggle with confirmation for enabling/disabling Seqera Compute.
- **Nextflow**

>   - Update `WfManifest` class in Tower to include the new `icon` property introduced in Nextflow.

Reviewer: In Tower?

Author: ```suggestion
  - Update `WfManifest` class to include the new `icon` property introduced in Nextflow.
```

  - Add a new migration file to add the new icon column into all the tables of entities that use the `WfManifest` class as @Embedded property.
- Audit log update: Publish a new *pipeline_updated* audit event whenever a user updates a pipeline.

> - AWS Cloud Provider compute environments: Support running the head job on AWS EC2 instances as an alternative to AWS Batch.

Reviewer: This one is confusing without additional context, or a link out to docs. Were we tasked with documenting this? This reads as a new compute environment type. And AWS Batch also runs on EC2 instances. @swampie what are we looking at here? A new CE type or just a new head job option inside AWS Batch CEs?

- Added `totalProcesses` count to workflow progress responses.
- Switch AWS Batch compute environment dependencies to AWS SDK v2.
- New workflow job monitoring collects and publishes Platform metrics.

> - Support user uploaded images to use as pipeline logo in the pipeline import and pipeline update forms.

Reviewer: ```suggestion
- Support user-uploaded images in the pipeline import and pipeline update forms, to be used as pipeline logos.
```
